src/compiler/scala/tools/nsc/profile/ExternalToolHook.java

```diff
@@ -0,0 +1,17 @@
+package scala.tools.nsc.profile;
+
+/**
```

Comment:
Can we simplify and make these docs clearer? There are typos, no capital letter and the actual meat of the docs is hidden under unnecessary words :smile:.